stack.yaml

```diff
@@ -58,6 +58,10 @@ packages:
     git: https://github.com/input-output-hk/cardano-crypto
     commit: 96adbd5aa9a906859deddf170f8762a9ed85c0c9
   extra-dep: true
+- location:
+    git: https://github.com/haskell-crypto/cryptonite-openssl
+    commit: fc6737d5f888ce039fc18442386c64050c3e7e05
```

Comment:
I needed to bump cryptonite-openssl in order to work with openssl-1.1.0e (latest openssl - there is no release yet)